UPGRADE.md

```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`

```

Comment:
I would add information about platform, e.g. "copy new `docker/conf/docker-compose.yml.dist` (be aware that there are different `dist` files for each platform so you need to copy the proper one) into your local..."

Reply:
For the same reason I would change it to "check changes in the `docker-compose.yml` template you used, a new container with Elasticsearch has to be added"